book.rb

```diff
@@ -0,0 +1,10 @@
+# Class representing a book.
+class Book
```

Comment:
- [ ] The Book class should include a method for adding rentals that represent the `many-to-many` relationship kindly consider creating a method that uses the Rentals itself to add rentals
e.g
```ruby
def add_rental(person, date)
  Rental.new(date, self, person)
end
```